tests/integration_tests/integration_test_config_example.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Yeah, nice idea

I've reworked test config in core-config manner

As for `GENERIC_CHAT_COMPLETION_DEPLOYMENTS` - is think that's easier to just stick to logic we're using right now: all unspecified deployments fall back to `GPT_TEXT_ONLY`

But to avoid mistakes, when somebody just forgot to include deployment into needed env variable, I've added information about deployment type to test id

Now it looks like this:

```
tests/integration_tests/test_chat_completion.py::test_chat_completion[image_in_custom_content/GPT4O__gpt_4o_2024_05_13/stream:on/tokens:800]
```